### PR TITLE
Snyk install jq if not installed

### PR DESCRIPTION
**Pull request by creydr**
Currently snyk tests fail, when jq is not installed
```
snyk installed to /tmp/snyk
1.1237.0
/bin/bash: line 18: jq: command not found
```
This PR addresses it and installs jq if not installed.

### CODE REVIEW

File: ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh [hunk 1]
```diff
@@ -13,6 +13,16 @@ chmod +x ${SNYK_DIR}/snyk
 echo snyk installed to ${SNYK_DIR}
 ${SNYK_DIR}/snyk --version
 
+# install jq if not installed
+if ! [ -x "$(command -v jq)" ]; then
+    JQ_DIR=/tmp/jq
```
Comment (bear-redhat) [inline ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh:18]: I think you can just put it to ${SNYK_DIR} for simplicity

Comment (creydr) [reply ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh]: updated

**Review by 2uasimojo — COMMENTED** [review]
:+1:

File: ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh [hunk 2]
```diff
@@ -13,6 +13,13 @@ chmod +x ${SNYK_DIR}/snyk
 echo snyk installed to ${SNYK_DIR}
 ${SNYK_DIR}/snyk --version
 
+# install jq if not installed
+if ! [ -x "$(command -v jq)" ]; then
+    curl -Lo ${SNYK_DIR}/jq "https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux-amd64"
```
Comment (cfergeau) [inline ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh:18]: For what it's worth, executing pre-built binaries downloaded from the net without knowing how/where they were built, if they were tampered with, ... could be problematic, especially when it's related to security.

Comment (creydr) [reply ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh:19]: @cfergeau good point. I added a check for the checksum. Or any better approaches?

Comment (cfergeau) [reply ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh:19]: Thanks! If this runs on fedora/RHEL, you can also `yum install jq`.

**Pull request merged**